docs/deployment/containers/docker.txt

```diff
@@ -147,18 +147,6 @@ the ``crate`` command::
 
    This will result in degraded performance.
 
-You can also start a single node without any bootstrap checks by passing the
```

Comment:
why are we dropping this?

Reply:
I couldn't find a way to disable bootstrap a node without checks. Neither could I find the setting in the docs. Thus I removed it.

Reply:
The discovery.type setting should still cause CrateDB to not enforce the bootstrap checks.

I agree with Naomi, this shouldn't be dropped